spec/graphql/language/parser_spec.rb

```diff
@@ -1,562 +1,7 @@
 require "spec_helper"
+require 'graphql/Language/parser_tests'
```

Comment:
Just to throw in my two cents, that's exactly the issue. Has bitten me quite a few times.